content/blogposts/2022/ranking-in-a-week.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'Ranking in a week'
+description: 'TODO'
```

Comment:
@iskyOS Could you please suggest a description?

Reply:
```suggestion
description: 'In just over a week, four developers designed and built a data pipeline utilizing Sourcegraph's wealth of source code data. The result is Sourcegraph's new search ranking system inspired by PageRank.'
```